src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -8048,7 +8048,7 @@ class C
 }
 ";
             CreateCompilationWithSpan(source, parseOptions: TestOptions.RegularPreview)
-                .VerifyEmitDiagnostics(
+                .VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](http://example.com/codeflow?start=17&length=17)

Most of these cases were changed from `VerifyDiagnostics` in the previous PR. This first commit changes those cases back, to ensure the errors are reported before emit.

Reply:
Using `VerifyDiagnostics()` to verify that expected errors are reported before emit. And also using `VerifyEmitDiagnostics()` for missing member tests to ensure emit handles missing members.

Reply:
The tests for missing members use `VerifyEmitDiagnostics()` since missing members may affect the synthesized string hash methods.